### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by lugard1**
# Description
In this project I:
- [x] Create a table named owners.
- [x] Create a table named species.
- [x] Modify the animals table.
- [x] Insert data into the owners table.
- [x] Insert data into the species table.
- [x] Modify inserted animals so it includes the species_id value.
- [x] Modify inserted animals to include owner information (owner_id).
- [x] Write queries (using JOIN) to answer the relative questions.
- [x] Use Git-flow.
- [x] Fix merge issues.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,24 @@ CREATE TABLE animals (
 
 Alter TABLE animals
   ADD COLUMN species varchar(100);
-  
\ No newline at end of file
+--------------------------------------------------------
+-- create table owners--
+CREATE TABLE owners (
+    id integer PRIMARY KEY,
```
Comment (alexander16108) [inline schema.sql:17]: Optional  - its best practice to add `IDENTITY(1,1)` to your database to increment items in your database to easily keep track of items in your db 😄

File: schema.sql [hunk 2]
```diff
@@ -11,4 +11,24 @@ CREATE TABLE animals (
...
+    full_name VARCHAR(50) NOT NULL,
+    age integer NOT NULL
+);
+
+-- create table species--
+
+CREATE TABLE species (
+    id integer PRIMARY KEY NOT NULL,
```
Comment (alexander16108) [inline schema.sql:25]: Optional  - its best practice to add `IDENTITY(1,1)` to your database to increment items in your database to easily keep track of items in your db 😄

**Review by alexander16108 — APPROVED** [review]
Hi @lugard1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
All linters are passing ✔️
Good commit history ✔️
Good PR description ✔️
Follow Gitflow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 2 above)
Comment (lugard1) [reply schema.sql:25]: Thanks.

File: schema.sql (hunk 1 above)
Comment (lugard1) [reply schema.sql:17]: Thanks for the heads up.

**Pull request merged**